File: client/Client/Network/Messages/Session/CreateMapMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Network.Messages.Session
{
    public class CreateMapMessage
    {
        public const string TYPE = "CreateMapMessage";
        private SessionDTO _sessionDTO;
        public SessionDTO sessionDTO { get => _sessionDTO; set => _sessionDTO = value; }

        public CreateMapMessage()
        {
        }

        public CreateMapMessage(SessionDTO sessionDTO)
        {
            this.sessionDTO = sessionDTO;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/CreateMapMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

import de.hsrm.mi.swtpro03.server.game.dtos.SessionDTO;

public class CreateMapMessage {

    public static final String TYPE = "CreateMapMessage";

    private SessionDTO sessionDTO;

    public SessionDTO getSessionDTO() {
        return sessionDTO;
    }

    public void setSessionDTO(SessionDTO sessionDTO) {
        this.sessionDTO = sessionDTO;
    }
}
